ios-target/View/Chat/ChatListTableViewCell.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ChatListTableViewCell.swift
+//  ios-target
+//
+//  Created by sol manini on 7/16/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+class ChatListTableViewCell: UITableViewCell {
+
+  @IBOutlet weak var userPictureImageView: UIImageView!
+  @IBOutlet weak var targetIconImageView: UIImageView!
+  @IBOutlet weak var userNameLabel: UILabel!
+  @IBOutlet weak var lastMessageLabel: UILabel!
+  @IBOutlet weak var notificationsLabel: UILabel!
+  
+ static var reuseIdentifier = "ChatListTableViewCell"
+  
+  var match: MatchListItem? {
+    didSet {
```

Comment:
You extract the code into a helper like `updateCell` so it makes a bit more sense when reading.